src/sentry/api/endpoints/project_release_files.py

```diff
@@ -33,28 +36,29 @@ def load_dist(results):
     # Dists are pretty uncommon.  In case they do appear load them now
     # as trying to join this on the DB does terrible things with large
     # offsets (it would otherwise generate a left outer join).
-    dists = dict.fromkeys(x.dist_id for x in results)
-    if not dists:
+    dist_map = {}
+    dist_ids = dict.fromkeys(x.dist_id for x in results)
+    if not dist_ids:
         return results
 
-    for dist in Distribution.objects.filter(pk__in=dists.keys()):
-        dists[dist.id] = dist
+    for d in Distribution.objects.filter(pk__in=dist_ids.keys()):
+        dist_map[d.id] = d
 
     for result in results:
         if result.dist_id is not None:
-            dist = dists.get(result.dist_id)
-            if dist is not None:
-                result.dist = dist
+            result_dist = dist_map.get(result.dist_id)
+            if result_dist is not None:
+                result.dist = result_dist
 
     return results
 
 
-class ReleaseFilesMixin:
+class ReleaseFilesMixin(BaseEndpointMixin):
```

Comment:
mixins aren't sound -- I usually convert these to free functions instead

Reply:
I can remove `BaseEndpointMixin`'s children across the codebase and delete it in a future PR 👍🏼